RELEASING.md

```diff
@@ -10,8 +10,15 @@ these tasks are run on master and can be patched at any time.
 
 ## Stable release
 
-To generate a new stable release you must follow the following steps:
+To generate a new stable release you can run:
 
+```bash
+./scripts/release.sh 1.0.0
+```
+
+**Note:** Please omit the `v` from the version to release 
```

Comment:
[np] Could we check and do this in the script? It's counter-intuitive as the examples below all show the v present.

Reply:
Thought about it but was being lazy, I'll do the full work